### DISCUSSION TITLE
Welcome to SFINCS Discussions!

### DISCUSSION THREAD

**Discussion by Leynse** [Deltares/SFINCS#118] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and the type of SFINCS modelling application you're after.

**Comment by jieAmple** [comment]
Hello everyone, I am Jie, a coastal engineer. I recently started exploring SFINCS as I found it very useful for later usage of floodadapt. I set up a case model for my region with all the dataset available online. 

One thing I could not figure out is when I added the infiltration file .scs into the model, the inland flooding seemed never being populated. The water level driven coastal flooding and precipitation induced inland flooding (without infiltration) is okay. 

I have attached the python code for the model setup and input data here. hope it is easy for the discussion if anyone has a clue~ many thanks

[ECP_2.ipynb](https://github.com/user-attachments/files/29193578/ECP_2.ipynb)
[ECP_60min.csv](https://github.com/user-attachments/files/29193614/ECP_60min.csv)
[gebco_sg.tif](https://github.com/user-attachments/files/29193617/gebco_sg.tif)
[sg_srtmGL1.tif](https://github.com/user-attachments/files/29193618/sg_srtmGL1.tif)
[sg_srtmGL1_filled.tif](https://github.com/user-attachments/files/29193619/sg_srtmGL1_filled.tif)
[singapore_data_catalog.yml](https://github.com/user-attachments/files/29193623/singapore_data_catalog.yml)

**Reply by roeldegoede** [reply]
Hello @jieAmple, thank you for your interest in SFINCS! Would you be able to provide me a bit more information on the SFINCS version that you used to run the simulation?

 It sounds to me like something we have seen before in the Mount Faber release, but that should be solved since this commit: https://github.com/Deltares/SFINCS/pull/254. As of last week, there is a new released version online, the Galibier release, which does include thad bugfix. Might be worth exploring!

Cheers,
Roel
